src/Utilities.ts

```diff
@@ -206,14 +206,23 @@ export class Utilities {
     public static generateTempPath(filePath: string, hashValue?: string): string {
```

Comment:
>generateTempPath [](start = 18, length = 16)

Will this code handle illegal chars in the directory name? Such as escaped spaces? 

[https://element/%20%subelementhello%20%/blah.cpp](https://element/%20%subelementhello%20%/blah.cpp)